lib/features/family/features/reflect/presentation/widgets/gratitude_selection_widget.dart

```diff
@@ -27,29 +26,25 @@ class GratitudeSelectionWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return FunScaffold(
       canPop: false,
-      appBar: FunTopAppBar.primary99(
-        title: uimodel.reporter.firstName!,
-        leading: Padding(
-          padding: const EdgeInsets.all(8),
-          child: SvgPicture.network(
-            uimodel.reporter.pictureURL!,
-            width: 36,
-            height: 36,
-          ),
-        ),
-        actions: const [
+      appBar: const FunTopAppBar(
+        title: 'Question 3',
```

Comment:
There's a minimum of 3 questions asked before this one so I expect this to be question 4. Also if there's now more than 3 interviewers this number will not be correct if hardcoded.

Reply:
This question is always going to be nr 3 > [Linear](https://linear.app/givt/issue/KIDS-1647/update-question-screens-in-gratitude-game)
"Game play should be Question 1 - Random, Question 2 - Random, Question 3 - What are you grateful for?, Question 4 - How have you been Generous?"